.github/workflows/check-licenses.yml

```
@@ -0,0 +1,26 @@
+name: Check licenses
+
+on:
+  pull_request:
+
+jobs:
+  check-licenses:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3.3.0
+      - uses: actions/setup-node@v3.6.0
+        with:
+          node-version: '16.19.0'
+          registry-url: 'https://npm.pkg.github.com'
+          scope: '@paritytech'
+      - name: Check the licenses
+        run: |
+          shopt -s globstar
+
+          npx @paritytech/license-scanner@0.0.3 scan \
+            --ensure-licenses Apache-2.0 \
+            --ensure-licenses GPL-3.0-only \
+            ./**/*.rs
```

Would it make sense/be possible to split this steps into two to have a better fail point?
```suggestion
      - name: Set the shell
        run: shopt -s globstar
      - name: Check the licenses
        run: |
          npx @paritytech/license-scanner@0.0.3 scan \
            --ensure-licenses Apache-2.0 \
            --ensure-licenses GPL-3.0-only \
            ./**/*.rs
```
I understand if it's not possible, I was just thinking that it would look tidier.

---

Maybe you could add the option to the user's `bash_profile` or similar in the step which then would be read in the second step, but I'm not sure if that's tidier.